lib/improver/utilities/cube_manipulation.py

```diff
@@ -76,10 +75,10 @@ def _associate_any_coordinate_with_master_coordinate(
 
     if not cube.coords(master_coord):
         msg = (
-            "The master coordinate for associating other " +
-            "coordinates with is not present: " +
-            "master_coord: {}, other coordinates: {}".format(
-                master_coord, coordinates))
+                "The master coordinate for associating other " +
+                "coordinates with is not present: " +
+                "master_coord: {}, other coordinates: {}".format(
+                    master_coord, coordinates))
```

Comment:
Why this formatting change?  Also, could you get rid of the `+` signs please - these aren't needed inside brackets.

Reply:
I must have inadvertently allowed PyCharm to reformat the file hence these formatting changes and a few others....I've reverted them and removed the `+` signs.